rest/data/test.go

```diff
@@ -76,3 +81,61 @@ func (tc *DBTestConnector) FindTestsByTaskId(opts FindTestsByTaskIdOpts) ([]test
 
 	return res, nil
 }
+
+func TestCountByTaskID(ctx context.Context, taskID string, execution int) (int, error) {
+	t, err := task.FindOneIdOldOrNew(taskID, execution)
```

Comment:
Will this guarantee that the task execution it gets back is the expected task execution? For example, if they pass execution=100, is it supposed to return the latest execution or error? Same for execution=0 - is it supposed to return the latest execution or the first execution?

Reply:
Ah I see what you are saying now, I am going to add a check that the returned task has the same execution otherwise return an error. I think it makes more sense for users to return an error if the execution does not exist rather than silently return the latest execution.